go.mod

```diff
@@ -64,6 +64,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
 	k8s.io/kubectl v0.18.4
 	k8s.io/utils v0.0.0-20200410111917-5770800c2500
+	k8s.io/kubernetes v1.13.4
```

Comment:
We try not to import the entire `kubernetes` as a module as it is not meant to be consumed this way and can create dependency issues. See https://github.com/golang/go/issues/32776#issuecomment-505607726.

This also seems to pin a very old version.

Reply:
Thanks very much for the suggestions; made change to use k8s.io/kubelet api instead.